scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -45,15 +49,63 @@ public static BuildTargetResult InitPublish(BuildTargetContext c)
         [Environment("PUBLISH_TO_AZURE_BLOB", "1", "true")] // This is set by CI systems
         public static BuildTargetResult Publish(BuildTargetContext c)
         {
+            if (ShouldPublishToLatest())
```

Comment:
If this fails, what action should we take? Is the idea that for the last build running this will be true?